rental.rb

```diff
@@ -0,0 +1,11 @@
+class Rental
+  attr_accessor :date
```

Comment:
- [ ] Kindly add a `person` and a `book` so that we are able to write and read them.

Reply:
Done, as advised